wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -768,7 +802,19 @@ Das folgende ist ein Beispiel, wie eine Tabelle aussehen sollte:
 
 ## Blockzitate
 
-Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren.
+Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren. Der Name der zitierten Person, der nach dem Zitat folgt, muss mit einem vorangestellten Geviertstrich (`—`) wie folgt geschrieben werden:
+
+```markdown
+> plz enjoy game
+—rrtyui
+```
+
+Wenn nicht das Originalzitat angeben wird, sondern eine Übersetzung, dann muss dies folgendermaßen gekennzeichnet werden:
+
+```markdown
+> bitte habt Spaß am Spiel
+—rrtyui (auf Deutsch übersetzt)
```

Comment:
Not sure on the usage of em dashes here. See my other review comment for more details.

Reply:
You'd keep the em dash (since it's a reference for English, as I've explained in the [other comment](https://github.com/ppy/osu-wiki/pull/6839#discussion_r800088559)), and put a blank line between the blockquote and the name.